docs/simplecard.md

```diff
@@ -0,0 +1,101 @@
+---
+id: simplecard
+title: Single-Element Payment Form
+---
+
+[Build a single-element payment form (Beta)](https://developer.squareup.com/docs/payment-form/payment-form-walkthrough#build-a-single-element-payment-form-beta)
+
+Using the `SimpleCard` element is an alternative to building a payment form if you don't need the flexibility of individually placed payment card fields. [Read more about customizing the appearance](https://developer.squareup.com/docs/payment-form/cookbook/customize-form-styles#customize-the-appearance-of-the-single-element-payment-form).
```

Comment:
Could we add something about needing to complete steps 1-5 here https://square.github.io/react-square-payment-form/docs/paymentform? I'm worried about people jumping to this page and getting stuck if they copy-paste.

Reply:
@stephtaylor would be very helpful to have that!